backend/ibutsu_server/__init__.py

```diff
@@ -40,7 +41,7 @@ def get_app(**extra_config):
 
     # Shortcut
     config = app.app.config
-    config.setdefault("BCRYPT_LOG_ROUNDS", 20)
+    config.setdefault("BCRYPT_LOG_ROUNDS", 12)
```

Comment:
Having the log rounds be `20` made it take around 45 - 60 s just to create a user. It also took around 45 - 60 s to check the password of the user.

Reply:
Yeah, thanks, I was going to change this :-)